tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -684,3 +698,80 @@ def sync_volumes(volumes, index, vol_options, sync_ttl):
             b_index += 1
         except KeyError:
             break
+
+
+def brick_status_alert(hostname):
+    # fetching brick details of disconnected node
+    bricks = NS._int.client.read(
+        "clusters/%s/Bricks/all/%s" % (
+            NS.tendrl_context.integration_id,
+            hostname
+        )
+    )
+    for brick_info in bricks.leaves:
+        try:
+            brick_path = brick_info.key
+            lock = None
+            lock = etcd.Lock(
+                NS._int.client,
+                brick_path[1:]
+            )
+            lock.acquire(
+                blocking=True,
+                lock_ttl=60
+            )
+            if lock.is_acquired:
+                brick = NS.gluster.objects.Brick(
+                    fqdn=hostname,
+                    brick_dir=brick_path.split("/")[-1]
+                ).load()
+                if brick.status == BRICK_STATUS[0]:
+                    # raise an alert for brick
+                    msg = ("Status of brick: %s "
+                           "under volume %s in cluster %s chan"
+                           "ged from %s to %s") % (
+                               brick.brick_path,
+                               brick.vol_name,
+                               NS.tendrl_context.integration_id,
+                               BRICK_STATUS[0],
```

Comment:
Also I would suggest to use constants for brick status `Started` and `Stopped` and not an array.

Reply:
i will change it